Dockerfile

```diff
@@ -19,5 +19,5 @@ RUN apt-get update && \
 
 RUN mkdir -p ~/.ssh/ && ssh-keyscan -t ed25519 github.com >> ~/.ssh/known_hosts
 
-RUN npm install -g npm@10.9.0
+RUN npm install -g npm@11.0.0
 RUN npm install -g spago@0.93.41 purescript@0.15.15 purescript-psa@0.9.0 grunt-cli@1.5.0 esbuild@0.24.0 purs-tidy@0.11.0
```

Comment:
Normally we update everything at once, I see e.g. esbuild has a bugfix release out... But I suppose it's not really related to the task at hand...

Reply:
Good call out, I'll add that!